synapseclient/client.py

```diff
@@ -393,16 +397,12 @@ def log_response(response: httpx.Response) -> None:
         }
         self.credentials = None
 
-        if not isinstance(debug, bool):
-            raise ValueError("debug must be set to a bool (either True or False)")
-        self.debug = debug
-
         self.silent = silent
         self._init_logger()  # initializes self.logger
 
         self.skip_checks = skip_checks
 
-        self.table_query_sleep = 2
+        self.table_query_sleep = 0.2  # Sleep for 200ms between table query retries
```

Comment:
Just out of curiosity, how does this interval get decided?

Reply:
I can't speak to how it was originally decided however for this value I based it off how long some queries actually needed to wait during schematic integration tests. I found that a delay of 0.2 seconds meant that only 1 wait cycle was needed before results were needed. Any shorter and several calls were wasted waiting for Synapse to finish providing the results, any longer and we were wasting time doing nothing.